### ISSUE TITLE
Adapt Example 11.6 using Issend+Cancel

### ISSUE THREAD

**Issue by devreal** [mpi-forum/mpi-issues#529]
# Problem

The standard contains an example (11.6) that demonstrates the use of `MPI_Cancel` with an active send request (`MPI_Issend`). As per https://github.com/mpi-forum/mpi-standard/pull/635#issuecomment-974281657, this example cannot easily be removed since it is used to describe other aspects of MPI. However, cancellation of send requests has been deprecated and will be removed eventually, so Example 11.7 needs to be adapted to be standard compliant once cancellation of send requests is officially removed. We might as well do that for MPI 4.1 before support for cancelling send requests is removed. 

# Proposal

None yet.

# Changes to the Text

Remove cancellation of send requests from Example 11.6 and replace them with a feature that is not deprecated.

# Impact on Implementations

None.

# Impact on Users

Less confusion from the use of a deprecated feature in examples.

# References and Pull Requests

https://github.com/mpi-forum/mpi-standard/pull/635 was an attempt to remove the example but the issue is more complex than that since it seems to be used to describe progress semantics (https://github.com/mpi-forum/mpi-standard/pull/635#issuecomment-974281657).

**Comment by wesbland** [comment]
This wasn't read at the December 2022 meeting. The last opportunity for MPI 4.1 is to have it ready at the March 2022 meeting (and it needs to "pass" the reading).

@bosilca / @RolfRabenseifner Are you (or is someone else) planning to push this forward or should we move it out of the plan for MPI 4.1?

**Comment by bosilca** [comment]
Now that canceling send requests is officially deprecated, we should remove this example as proposed in my previous patch (mpi-forum/mpi-standard#635). I do not intend to do more than rebasing the branch.

**Comment by wesbland** [comment]
Fine with me. Feel free to reopen that PR and have it close this issue.

**Comment by RolfRabenseifner** [comment]
No, but this example has a notice that the cancel for a send request is deprecated. Deprecation means it is still part of the MPI standard. And the rules for implementors for handling deprecated send requests are not changed; they are documented with this example and the following AtoI.

**Comment by schulzm** [comment]
I agree with Rolf on keeping it, but shouldn't the example be moved to the deprecated chapter?
